src/devtools/Explorer.tsx

```diff
@@ -45,43 +52,81 @@ export const Expander = ({ expanded, style = {}, ...rest }) => (
   </span>
 )
 
-const DefaultRenderer = ({
-  handleEntry,
+type Entry = {
+  label: string
+}
+
+type RendererProps = {
+  HandleEntry: HandleEntryComponent
+  label?: string
+  value?: any
+  subEntries?: Entry[]
+  subEntryPages?: Entry[][]
+  type?: string
+  expanded?: boolean
+  toggleExpanded: () => void
+  pageSize: number
```

Comment:
it looks like we could make most (all?) of these props mandatory, as we always pass them in (there is only one usage)

Reply:
Remove optional for all except `label` 3b4aeef4bfa5ed8b3838593cc9558b39785bb4fc